build.gradle

```diff
@@ -3,7 +3,7 @@ import java.time.LocalDate
 
 plugins {
     id 'java'
-    id 'org.springframework.boot' version '2.7.5'
+    id 'org.springframework.boot' version '2.7.6'
```

Comment:
I think we may need to revert it back to 2.7.5. I think we had some issues with this upgrade last time. Will see.